Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG BASE_IMAGE=ubuntu:22.04
+FROM ${BASE_IMAGE} AS build-stage
+
+ARG PJ_TAG=3.9.2
+
+###############################################################################
+# Install general dependencies.
+###############################################################################
+RUN apt update && apt install -y build-essential cmake git 
+
+###############################################################################
+# Compile PlotJuggler
+###############################################################################
+WORKDIR /plotjuggler_ws/src
+RUN git clone --branch ${PJ_TAG} https://github.com/facontidavide/PlotJuggler.git PlotJuggler
```

Comment:
add --depth 1 --no-single-branch so it don't get the full history but just the branch latest tag. Again to gain some space.